love/main.lua

```diff
@@ -198,7 +207,10 @@ function love.load(args)
   local small_font = love.graphics.newImageFont("images/5x3font_bm.png", "!\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~ ", 1)
   love.graphics.setFont(small_font)
 
-  LuaGB.game_screen_imagedata = love.image.newImageData(256, 256)
```

Comment:
powers of two are generally faster to work with, especially on older graphics cards.